<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AUtoClickMClass</class>
 <widget class="QMainWindow" name="AUtoClickMClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>幼圆</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>AUtoClickM</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../autoClick/ac.png</normaloff>../../../../autoClick/ac.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 170, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSpinBox" name="AutoClickTimes">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>自动点击次数：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IsAutoClick">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>连续自动点击</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 127);</string>
    </property>
    <property name="text">
     <string>右键关闭</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>自动点击间隔：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AutoClickSleepTime">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>左键+shift开始点击</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>-------------------------------------------------</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>录制点击：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IsAutoClick_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>连续自动点击</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>F7录制位置  F8开始点击 F9关闭点击</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="AutoClickTime1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="singleStep">
     <number>3</number>
    </property>
    <property name="value">
     <number>40</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>次</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>ms</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="clickTimesE">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>手动间隔</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="ClickTable">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>601</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>5</number>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>107</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>20</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>20</number>
    </attribute>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string>计数</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>x坐标</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>y坐标</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>延时</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>其他</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="path">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
     </font>
    </property>
    <property name="text">
     <string>选择cok文件路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LiuLan">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ReadCokFile">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>读取cok</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>60</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>显示cok</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="debugText">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>291</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="markdown">
     <string/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:7pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'SimSun'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清除</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>AutoClickTimes</zorder>
   <zorder>label</zorder>
   <zorder>IsAutoClick</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>AutoClickSleepTime</zorder>
   <zorder>label_4</zorder>
   <zorder>label_6</zorder>
   <zorder>IsAutoClick_2</zorder>
   <zorder>label_10</zorder>
   <zorder>AutoClickTime1</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>label_11</zorder>
   <zorder>clickTimesE</zorder>
   <zorder>ClickTable</zorder>
   <zorder>path</zorder>
   <zorder>label_7</zorder>
   <zorder>LiuLan</zorder>
   <zorder>ReadCokFile</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>debugText</zorder>
   <zorder>remove</zorder>
   <zorder>clear</zorder>
   <zorder>save</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AUtoClickM.qrc"/>
 </resources>
 <connections/>
</ui>
